src/html.coffee

```diff
@@ -181,13 +182,14 @@ htmlModule.list = ({parent, data, label, link, class:listClass, selection, selec
               d3.event.stopPropagation() if icon.onLeave?(val, i) is false
               return
             ).append('div')
+              .merge(el)
               .attr('class', 'icon')
 
           return
         return
 
       sel = ul.selectAll('li').data(_data)
-      a = sel.enter().append('li').append('a')
+      a = sel.enter().append('li').merge(sel).append('a').merge(sel)
 
       addIcons a, 'left'
       a.append('span')
```

Comment:
Does a `merge` need to be called here too? I'm not entirely sure when we need to call it, but from the change log it says `selection.append no longer merges entering nodes into the update selection` which is why I'm assuming it's added after every append.

Reply:
I'm not 100% sure either but I've been adding a `merge` after every `append`... just missed this one, I'll add it here too

Reply:
I take it back, I don't think we need it here.  `merge` is used when we `append` new elements and then want to perform the same operations (e.g. set `attr`s) on both the old and the new elements.  Here, we aren't performing any operations on the appended elements or assigning the selection to a variable; we're just appending elements and then moving on.  So there's no need to `merge`.